instrumentation/build.gradle.kts

```diff
@@ -29,6 +28,18 @@ android {
         }
     }
 
+    androidComponents {
+        onVariants {
+            if (it.buildType == "release") { // The one we choose to release
+                project.tasks.register("createReleaseBuild", Copy::class) {
```

Comment:
Why can't we use ./gradlew assembleRelease to create the release variant of app instead of creating custom task

Reply:
Yeah, @LikeTheSalad added a snippet so we can tag onto `assembleRelease` and therefore `assemble`...so I've included that. Thanks!